dair_pll/multibody_learnable_system.py

```diff
@@ -339,20 +275,19 @@ def forward_dynamics(self,
                                      contact_filter.transpose(-1,
                                                               -2).int()).bool()
 
-        n = n_contacts * 3
-        reorder_mat = torch.zeros((n, n))
-        for i in range(n_contacts):
-            reorder_mat[i][3 * i + 2] = 1
-            reorder_mat[n_contacts + 2 * i][3 * i] = 1
-            reorder_mat[n_contacts + 2 * i + 1][3 * i + 1] = 1
-        reorder_mat = reorder_mat.reshape(
-            (1,) * (delassus.dim() - 2) + reorder_mat.shape).expand(
-            delassus.shape)
-
-
+        n_force = n_contacts * 3
```

Comment:
This comment isn't due to new changes, but an artifact of the original code:  This `reorder_mat` construction is repeated in two places.  Might be useful to wrap it into one helper function and call just that in two places.